laia/scripts/htr/train_ctc.py

```diff
@@ -79,6 +79,7 @@ def run(
         tr_txt_table=tr_txt_table,
         va_txt_table=va_txt_table,
         batch_size=data.batch_size,
+        min_valid_size=model.get_min_valid_image_size(),
```

Comment:
You do not set any value for `max_search_size`. We should avoid *magic* values like `128`.
Usually we don't want to pad every images of our dataset. The maximum search value should be equal to the maximum width of our images. To quickly find this value, instead of using Pillow you could use [imagesize](https://pypi.org/project/imagesize/) which looks up image sizes quickly.

Reply:
This value should be set to None if we don't fix the height of the images.